src/client/balance.rs

```diff
@@ -0,0 +1,11 @@
+use clap::ArgEnum;
+
+use crate::{currency::Currency, error::BundlrError, Bundlr};
+
+pub async fn run_balance(url: &str, address: &str, currency: &str) -> Result<String, BundlrError> {
```

Comment:
I find the function name a bit odd, why says `run`? Why not `get`?

Reply:
Balance is tokyo work, in this context, not a Api fetch or something similar